File: api/Models/BlogContent.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ByzmoApi.Models
{
    public class BlogContent
    {
        public int Id { get; set; }
        public int BlogId { get; set; }
        public string HeaderText { get; set; }
        public string Description { get; set; }
        public int TypeId { get; set; }
        public int SortId { get; set; }
        public IEnumerable<BlogImage> ContentImages { get; set; }
        public bool IsDeleted { get; set; }
        public bool IsNew { get; set; }
        public string VideoLink { get; set; }
    }
}

File: web/src/app/classes/blog-content.ts
import { FileMapper } from './file-mapper';

export class BlogContent {
    id: number;
    blogId: number;
    headerText: string;
    description: string;
    contentImages: Array<FileMapper>;
    typeId: number;
    type: string;
    sortId: number;
    isDeleted: boolean;
    isNew: boolean;
    videoLink: string;
    embeddedUrl: any;
    constructor() {
        this.contentImages = new Array<FileMapper>();
        this.videoLink = '';
    }
}
